src/lib/cli/exit.js

```diff
@@ -13,6 +13,8 @@ import chalk from 'chalk';
  */
 
 export function withError( message: string ) {
-	console.log( chalk.red( message.toString().replace( /^(Error: )*/, 'Error: ' ) ) );
+	let updatedMessage = message;
```

Comment:
Whats the purpose of this assignment here?

Reply:
Oh, I changed my mind halfway through :)